objectbox/lib/src/observable.dart

```diff
@@ -61,20 +62,25 @@ class _Observable {
       _Observable._anyObserver.containsKey(store.ptr.address);
 }
 
+/// Streamable adds stream support to queries. The stream reruns the query
+/// whenever there's a change in any of the object in the queried Entity
```

Comment:
```suggestion
/// whenever there's a change in any of the objects in the queried Entity
```
Also maybe `Box` instead of `Entity`? Then also below for `findStream()`.